app-doc/eclass-manpages/files/eclass-to-manpage.awk

```diff
@@ -90,10 +90,9 @@ function xfail(text) {
 	_stderr_msg(text, "error (ignoring)")
 	exit(77)
 }
-
```

Comment:
Why did you eat this line?

Reply:
Oops, that's where I had the original `eat_eapis` function. Functions above (short ones) aren't separated by a newline, but functions below (long ones) are. I must've looked example from above :P 

thanks!

Reply:
I think they're actually grouped, and groups are separated.